anthos-vmruntime/README.md

```diff
@@ -159,33 +161,47 @@ create the VM using `kubectl apply -f pos-vm.yaml`.
 kubectl virt create vm pos-vm \
 --boot-disk-size=80Gi \
 --memory=4Gi \
---cpu=2 \
+--vcpus=2 \
```

Comment:
Can you fix the formatting error in line 156?
`[/anthos-vmruntime/pos-vm.yaml](/pos-vm.yaml)` must be `[/anthos-vmruntime/pos-vm.yaml](pos-vm.yaml)`

Reply:
Done!